CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+<a name="5.0.0"></a>
+# 5.0.0 (Unreleased)
+
+### Breaking Changes
+* **build** Terra-Components is now fully AOT compatible. 
+Terra-Components now complies with the angular package format and code autocompletion should work fine in your angular app.
+* **styles** Style files styles/styles.scss & styles/icons.scss & styles/theme-loader.scss are now .css files rather than .scss files.
```

Comment:
```suggestion
* **styles** Style files such as styles/styles.scss, styles/icons.scss and styles/theme-loader.scss are now .css files rather than .scss files.
```